src/Shouldly/ShouldlyExtensionMethods/ShouldBe/StringStartEndTestExtensions.cs

```diff
@@ -19,19 +25,28 @@ public static void ShouldEndWith([NotNull] this string? actual, string expected,
     }
 
     /*** ShouldNotStartWith ***/
```

Comment:
This doesn't have to be part of this PR, but feel free to delete these `/*** coments ***/`

Reply:
Will do in a separate PR so as to not create merge conflicts and for better PR separation.